doc/news/changes/minor/20191016MartinKronbichlerPeterMunchMichalWichrowski

```diff
@@ -0,0 +1,4 @@
+New: Provide cell iterators to interior and exterior cells of
+a macro face by MatrixFree.
```

Comment:
If you write these entries to say "Provide cell iterators to interior and exterior cells of
a macro face through the function MatrixFree::get_face_iterator().", then doxygen will automatically link to the new function.